File: Em C/1074.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int N,X,i;
	
	scanf("%d",&N);
	for(i=0;i<N;i++){
		scanf("%d",&X);
		if(X<0 && X%2==0){
			printf("EVEN NEGATIVE\n");
		}else{
			if(X<0 && (X%2==1 || X%2==-1)){
			   printf("ODD NEGATIVE\n");	
			}
			if(X>0 && X%2==0){
			printf("EVEN POSITIVE\n");
		    }
		    if(X>0 && X%2==1){
			printf("ODD POSITIVE\n");
		    }
		    if(X==0){
			printf("NULL\n");
		    }
		}
	}
    return 0;
}

File: Em Ruby/1074.rb
N=gets.chomp.to_i
contador=1
	while contador <= N do
		X = gets.chomp.to_i
		  if X<0 and X%2==0
			  puts "EVEN NEGATIVE"
		  end
			if X<0 and (X%2==1 or X%2==-1)
			   puts "ODD NEGATIVE"	
			end
			if X>0 and X%2==0
			  puts "EVEN POSITIVE"
		  end  
		  if X>0 and X%2==1
			  puts "ODD POSITIVE"
		  end  
		  if(X==0)
			  puts "NULL"
		  end
      contador +=1
  end      
